Cargo.toml

```diff
@@ -1,21 +1,17 @@
 [package]
 name = "poseidon252"
-version = "0.1.0"
+version = "0.2.0"
 authors = [
   "zer0 <matteo@dusk.network>"
```

Comment:
Add yourself to the list of authors, as well as Victor.

Reply:
Shouldn't we wait for #11 to land to then make the bump?